### PR TITLE
Cleanup post Compose migration

### PR DESCRIPTION
**Pull request by BottleRocket-Colin**
Cleared out styles and other resource, removed unused libraries.

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -144,24 +144,23 @@ dependencies {
...
     lifecycleDependencies()
     navigationDependencies()
 
-    koinDependencies()
-
     coreLibraryDesugaringDependencies()
 
     // UI
-    groupieDependencies()
+    // groupieDependencies()
```
Comment (jaredstockton) [inline app/build.gradle.kts]: @BottleRocket-Colin What's the reason for commenting out vs removing? Seems like removing would be the way to go :)

I'll remove this in a review commit.

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 2]
```diff
@@ -195,13 +189,9 @@ private object Libraries {
...
     const val COMPOSE_UI_TEST = "androidx.compose.ui:ui-test-junit4:$COMPOSE_VERSION"
 
-    // https://developer.android.com/jetpack/androidx/releases/constraintlayout
-    const val CONSTRAINT_LAYOUT = "androidx.constraintlayout:constraintlayout:2.1.3"
 
     // Navigation
     // https://developer.android.com/jetpack/androidx/releases/navigation
-    const val NAVIGATION_FRAGMENT_KTX = "androidx.navigation:navigation-fragment-ktx:$NAVIGATION_VERSION"
-    const val NAVIGATION_UI_KTX = "androidx.navigation:navigation-ui-ktx:$NAVIGATION_VERSION"
     const val NAVIGATION_COMPOSE = "androidx.navigation:navigation-compose:2.4.0-rc01"
```
Comment (jaredstockton) [inline buildSrc/src/main/kotlin/Dependencies.kt:190]: @BottleRocket-Colin It looks like all of the navigation safe args code can be remove as well:

* `NAVIGATION_VERSION` (in this file)
* `NAVIGATION_SAFE_ARGS_GRADLE` (in this file)
* `classpath(Config.BuildScriptPlugins.NAVIGATION_SAFE_ARGS_GRADLE)` (in project `build.gradle`)

I'll remove this in a review commit.

File: app/build.gradle.kts [hunk 3]
```diff
@@ -144,24 +144,23 @@ dependencies {
...
 
+    // Koin DI
+    koinDependencies()
+
     // AndroidX
     composeDependencies()
     accompanistDependencies()
     appCompatDependencies()
     activityDependencies()
-    fragmentDependencies()
```
Comment (jaredstockton) [inline app/build.gradle.kts:152]: @BottleRocket-Colin It looks like a few additional fragment related things can be removed:

* `FRAGMENT_VERSION`
* `ANDROIDX_FRAGMENT_TEST`
* `fragmentTestingDependencies` (definition and usage in app `build.gradle.kts`)

I'll remove this in a review commit.

**Review by jaredstockton — COMMENTED** [review]
It's coming along! I'll push a review commit to address some of the comments I've left as well as some additional cleanup items I've noticed (mainly unused classes, interfaces, and tests)

**Comment by jaredstockton** [comment]
@BottleRocket-Colin Please take a look at the review commit when you get a chance.

Additionally, I think we should be able to remove `kapt` and only use `ksp` at the moment. Databinding was the only thing in the project that required `kapt` and it is no longer used. Thoughts?

**Comment by BottleRocket-Colin** [comment]
> @BottleRocket-Colin Please take a look at the review commit when you get a chance.
> 
> Additionally, I think we should be able to remove `kapt` and only use `ksp` at the moment. Databinding was the only thing in the project that required `kapt` and it is no longer used. Thoughts?

Makes sense and definitely worth trying to switch to the newer utility

**Comment by jaredstockton** [comment]
@BottleRocket-Colin I did notice that there were some databinding related values in the `gradle.properties` file so just removed them in another review commit

**Comment by jaredstockton** [comment]
> Makes sense and definitely worth trying to switch to the newer utility

`KSP` is already integrated (`moshi` is the only dependency using it at the moment). Looking at the [libraries that support ksp](https://kotlinlang.org/docs/ksp-overview.html#supported-libraries) (as well as kapt), the only ones that might create a need to keep `kapt` is if we ended up using `Dagger` and/or `Hilt`, which I'm not expecting to be the case for the architecture demo.

If you agree, I'll create another PR to remove `kapt`.

**Review by jaredstockton — APPROVED** [review]

**Pull request merged**